src/main/sensors/esc_sensor.c

```diff
@@ -1577,13 +1843,19 @@ bool INIT_CODE escSensorInit(void)
         case ESC_SENSOR_PROTO_APD:
             baudrate = 115200;
             break;
+        case ESC_SENSOR_PROTO_OPENYGE:
+            callback = oygeDataReceive;
+            baudrate = 115200;
+            mode = MODE_RXTX;
+            options |= SERIAL_BIDIR;
```

Comment:
What is this for?
It would imply the telemetry must be connected to the TX pin on the FC.
The other telemetry protocols are connected to the RX pin.

Reply:
Needed for bi-directional (single wire serial communication) between the FC and connected devices with a single signal wire connection - both devices are connected by TX pin only allowing the use of 2 lead (or 3 lead connectors if +ve is needed).

Bi-dir is required for ESC setup/programming (if supported by the ESC).
Coming soon for YGE - I'd like to get Scorpion and OMP done too
![image](https://github.com/rotorflight/rotorflight-firmware/assets/4014433/e8a1a4a8-c4da-4d95-8e4f-d03a8b93c8c4)

Doing it now appears to have no downsides vs later which would require telemetry cablers in use to be modified (or resources remapped)

Reply:
The downside is that the release won't have pin swapping in the Configurator, or in the serial drivers.
Users would need to have a different configuration for YGE telemetry setup, likely with "complex" CLI commands.
As the Tx is not used right now, I suggest you keep it Rx only for the forthcoming release, so that it works the same as the other protocols.
The two-way development can happen after the release.

Reply:
Agreed, RX for initial release.
RX / TX matter will be much easier deal with if/when pin remapping comes to the configurator
Thanks